master/shapes.html

```diff
@@ -201,7 +201,7 @@ <h2 id="CircleElement">The <span class="element-name">'circle'</span> element</h
 
 <p>
   The <a>'r'</a> attribute defines the radius of the circle.
-  A negative value for either property is illegal and <a href="https://www.w3.org/TR/CSS21/syndata.html#parsing-errors">must be ignored as a parsing error</a>.
+  A negative value for either property is illegal and <a href="https://www.w3.org/TR/CSS21/syndata.html#parsing-errors">must be ignored as a parsing error</a>. A computed value of zero disables rendering of the element.
```

Comment:
```suggestion
  A negative value is illegal and <a href="https://www.w3.org/TR/CSS21/syndata.html#parsing-errors">must be ignored as a parsing error</a>.
  A computed value of zero disables rendering of the element.
```

Fix another existing copy-paste problem with the text. (Remove "for either property" because we're only talking about one property here.)

Also: add a semantic line break before the new text.